docs/notebooks/Training_visualizations.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
Please add an example with logger="shell" in notebook (and show logging output in notebook)